### DISCUSSION TITLE
How to change 'OltVendorId'

### DISCUSSION THREAD

**Discussion by SMSegura** [Anime4000/RTL960x#48] [Q&A] [answered]
Hello.
My Internet provider is Movistar, and it uses the VLans 6:1 for Data and 3:4 for VoIP. With these VLans, I perfectly raise Data and Voice with a ZIZA OP151S module.
I acquired the DFP-34X-2C2 module and when I execute the command:

'# omcicli mib get 84
XXXXXXXXXXXXXXXXXXXXXXX
VlanTagFilterData
XXXXXXXXXXXXXXXXXXXXXXX
' ==================
EntityID: 0x1102
FilterTbl[0]: PRI 0,CFI 0, VID 832
FwdOp:  0x10
NumOfEntries: 1
' ==================
' ==================
EntiryID: 0x1103
FilterTbl[0]: PRI 0,CFI 0, VID 3
FwdOp:  0x10
NumOfEntries: 1
' ==================
' ==================
EntiryID: 0x110b
FilterTbl[0]: PRI 0,CFI 0, VID 3
FwdOp:  0x10
NumOfEntries: 1
' ==================

The Data VLan is 832, which is the one used by Orange. I configure Data with the 832 but I can't get a stable connection.

I have seen that if I run:

'# omcicli mib get 131
XXXXXXXXXXXXXXXXXXXXXXX
OltG
XXXXXXXXXXXXXXXXXXXXXXX
' ==================
EntityId: 0x00
OltVendorId: 0x414c434c
EquipId:
Version:
ToDInfo:
        Sequence number of GEM superframe: 0x0
        Timestamp: secs 0, nanosecs 0
' ==================

the OltVersionId is (0x414c434c) ALCL I am trying to change it to HWTC with the command:
  'omcicli mib set 131, OltG, OltVendorId, 0x48575443'
 but it doesn't save the changes.

Am I missing something to do?

I'm trying to make the OLT believe that I'm connecting to HWTC.

thanks for any help

**Comment by vuducdong** [comment]
@SMSegura  you mean, right?

![image](https://user-images.githubusercontent.com/74129390/177782956-98c2efba-b8fa-4d7f-be51-a6ca02d5117e.png)

flash set PON_VENDOR_ID HWTC

**Comment by SMSegura** [comment, marked as answer]
Thanks for answering.
Anime400 answers me the following:

OltVendorId tell which OLT are you connected to, similar to Huawei MA5671a OpenWRT Firmware show far end OLT brand Id.
You only can change PON_VENDOR_ID to HWTC
